tests/test_ru.py

```diff
@@ -127,3 +127,15 @@ def test_to_currency(self):
                       cents=False, currency='EUR'),
             "тридцать восемь евро и 40 центов"
         )
+        self.assertEqual(
+            num2words('1230.56', lang='ru', to='currency', currency='USD'),
+            'одна тысяча двести тридцать долларов, пятьдесят шесть центов'
+        )
+        self.assertEqual(
+            num2words('1231.56', lang='ru', to='currency', currency='USD'),
+            'одна тысяча двести тридцать один доллар, пятьдесят шесть центов'
+        )
+        self.assertEqual(
+            num2words('1234.56', lang='ru', to='currency', currency='USD'),
+            'одна тысяча двести тридцать четыре доллара, пятьдесят шесть центов'
```

Comment:
The CI is failing due to this line being bigger than 79 characters. Splitting it should fix the problem.